code/game/objects/items/devices/paicard.dm

```diff
@@ -55,16 +57,19 @@
 	user.set_machine(src)
 	ui_interact(user)
 
+/// Opens the TGUI window
```

Comment:
These docs for the overrides are pretty useless, the ones we want are base vars and base procs.